API_flask.py

```diff
@@ -0,0 +1,85 @@
+import checkmanga
+from flask import Flask, request, jsonify
+app = Flask(__name__)
+
+checkmangautil = checkmanga.CheckMangaUtil('mycheckmanga.db')
+
+def validate_input_json_keys(request_json, required_keys):
+	for key in required_keys:
+		if key not in request_json.keys():
+			return key
+	return True
+
+def process_json_wrapper(request, func, route_name, required_keys):	
+	req_json = request.get_json()
+	if not req_json: 
+		return {"Failed": "Request JSON was empty."}
+	validate = validate_input_json_keys(req_json, required_keys)
+	if validate != True:
+		return {"Failed": "Failed JSON request had missing key: {0}".format(validate)}
+	function_result=func(req_json)
+	if function_result: 
+		return {"Success": "Successful call to {0} with parameters: {1}".format(route_name, str(req_json))}
+	else:
+		return {"Failed": "API call failed because request JSON had invalid value."}
+	
+@app.route("/")
+def hello():
+	return "Welcome to the CheckManga API!"
+
+@app.route('/api/addmanga/', methods=['POST'])
+def pass_to_add_manga():
+	req_keys = ["Title", "Site", "Status", "Url", "CoverImageUrl"]
+	status_msg = process_json_wrapper(request, checkmangautil.add_manga, "add_manga", req_keys)
+	return jsonify(status_msg)
+
+@app.route('/api/deletemanga/', methods=['POST'])
```

Comment:
delete? also API endpoints should not be named `delete_` or `add_` etc
just call it `manga` or `site`